new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAddCluster.java

```diff
@@ -107,7 +107,7 @@
   private static String dockerConfigJson = "";
 
   private static int replicaCount = 2;
-  private static final String domainUid = "miiaddcluster";
+  private static final String domainUid = "mii-add-cluster";
```

Comment:
this is for line 125. I am not able to comment there. Can you add a *?

Reply:
done